### PR TITLE
Add an Avatar component

### PR DESCRIPTION
**Pull request by jimmynotjim**
Adds an Avatar component that will automatically fallback to using the user's initials if a url for a background image isn't passed.

### CODE REVIEW

**Comment by jimmynotjim** [comment]
Adding @ant-dot-comm for review.

**Comment by jimmynotjim** [comment]
Updated to make it more in line with the `Button` work and include tests.

Also adding @christygurga for review.

File: src/shared/components/atoms/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            ${avatarURL}/38,
+            ${avatarURL}/76 2x,
+            ${avatarURL}/114 3x,
+            ${avatarURL}/152 4x
+        `;
+
+        if (avatarURL && avatarName) {
+            return(
+                <img
+                    alt={avatarName}
```
Comment (christygurga) [inline src/shared/components/atoms/Avatar/Avatar.tsx]: Could we make this truly optional, so if the name is next to the avatar, we can have an empty alt?

Comment (jimmynotjim) [reply src/shared/components/atoms/Avatar/Avatar.tsx]: Good call, updated.

File: src/shared/components/atoms/Button/Button.scss [hunk 2]
```diff
@@ -13,8 +13,8 @@
     padding: $global-spacing-half $global-spacing;
 
     &:hover {
-        background-color: $color-secondary;
-        color: $white;
+        background-color: $color-dark-primary;
```
Comment (christygurga) [inline src/shared/components/atoms/Button/Button.scss]: Is this the WCAG AA green? Let's talk about the naming for these

**Pull request merged**